tests/actions.go

```diff
@@ -363,6 +365,36 @@ func (oi *OperatorConfig) OperatorHelmSetString(m map[string]string) string {
 	return strings.Join(arr, ",")
 }
 
+func (oa *operatorActions) runKubectlOrDie(args ...string) string {
+	cmd := "kubectl"
+	glog.Infof("Running '%s %s'", cmd, strings.Join(args, " "))
+	out, err := exec.Command(cmd, args...).CombinedOutput()
+	if err != nil {
+		glog.Fatalf("Failed to run '%s %s'\nCombined output: %q\nError: %v", cmd, strings.Join(args, " "), string(out), err)
+	}
+	glog.Infof("Combined output: %q", string(out))
+	return string(out)
+}
+
+func (oa *operatorActions) CleanCRDOrDie() {
+	oa.runKubectlOrDie("delete", "crds", "--all")
+}
+
+// InstallCRDOrDie install CRDs and wait for them to be established in Kubernetes.
+func (oa *operatorActions) InstallCRDOrDie() {
+	oa.runKubectlOrDie("apply", "-f", oa.manifestPath("e2e/crd.yaml"))
+	out := oa.runKubectlOrDie([]string{"get", "crds", "--no-headers", `-ojsonpath={range .items[*]}{.metadata.name}{" "}{end}`}...)
+	waitArgs := []string{"wait", "--for=condition=Established"}
```

Comment:
The default wait timeout is 30s, I think we should specify a longer duration.

Reply:
It's enough to establish CRDS in kube-apiserver, normally it will be ready in a few seconds. We can increase it when we see failures in the future.